model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -101,11 +101,6 @@ data class AnalyzerResult(
             }
         }
 
-    /**
-     * True if there were any issues during the analysis, false otherwise.
-     */
-    val hasIssues by lazy { getAllIssues().isNotEmpty() }
```

Comment:
Commit message: Could you refer to @mnonnenmacher's hint that this was originally introduced / serialized for more convenient parsing of the ORT result file in a Jenkins job, but that this is not needed anymore due to the proper exit code handling that we have by now?

Also from the wording, I probably wouldn't say that the property "holds redundant information", but that the convenience it offered is not needed anymore.

Reply:
The same can go via copy & paste to the other commit messages then.

Reply:
I found it with `grep` because it was the last file left with `has_issues` in it.